managedplugin/download.go

```diff
@@ -154,6 +154,22 @@ func doDownloadPluginFromHub(ctx context.Context, c *cloudquery_api.ClientWithRe
 	case http.StatusUnauthorized:
 		return fmt.Errorf("unauthorized. Try logging in via `cloudquery login`")
 	case http.StatusNotFound:
+		// See if the plugin exists, but not the version.
+		pvw, err := NewPluginVersionWarner(logger, ops.AuthToken)
+		if err != nil {
+			return fmt.Errorf("failed to create plugin version warner: %w", err)
```

Comment:
In this case might be better to keep the [original error](https://github.com/cloudquery/plugin-pb-go/pull/423/files#diff-65f60ea2b094fb37ad7333772b6c3708cd026ed6729984bc77f8878de444bc65R173) that instructs users to run `cloudquery login`

Reply:
Done